readme.md

```diff
@@ -1,4 +1,8 @@
 # Nimbus
 Nimbus is a .NET client library to add an easy to develop against experience against the Azure Service Bus brought to you by Damian Maclennan and Andrew Harcourt
 
+This repository is forked from [Nimbus API](https://github.com/NimbusAPI), bringing additional functionality on top of the functionality provided by Nimbus. This forkmaster branch is the default branch, and is still based on [Nimbus V2](https://github.com/NimbusAPI/Nimbus_v2), since version 3 has introduced changes such as at-most-once delivery guarantees, as opposed to at-least-once delivery guarantees.
```

Comment:
I suggest rather than listing, we add a filter link to PRs into Nimbus from the Aqovia fork. We raise an upstream PR for every enhancement so this will then be a living list (and a checklist for eventual closure of the fork).